sigle/src/modules/settings/SettingsForm.tsx

```diff
@@ -124,6 +135,7 @@ interface SettingsFormProps {
 export const SettingsForm = ({ settings, username }: SettingsFormProps) => {
   const queryClient = useQueryClient();
   const { user } = useAuth();
+  const [unsavedChanges, setUnsavedChanges] = useState(false);
```

Comment:
instead of using this react state, could we use the dirty formik prop? https://formik.org/docs/api/formik#dirty-boolean

Reply:
ah I wasn't aware of this property, neat thanks!